src/components/Pill/story.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import Pill from './';
+import Readme from './README.md';
+
+function handleClick(id) {
+  alert(`clicked ${id}`);
+}
+storiesOf('Uncategorized/Pill', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
```

Comment:
The typical pattern for these stories uses `Overview` as a place to use Storybook knobs to manipulate the component in whatever was possible to see various possibilities. What's defined here is a bit more what we've gone with `Examples`.